docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: "3.7"
+
+# Will use the hostname when talking between services on the same network
+services:
+  ubkg-api:
+    build:
+      context: .
+    hostname: ubkg-api
+    container_name: ubkg-api
+    # Map host machine port 6666 to container port 8080
+    # Only root can listen on ports below 1024, we use higher-numbered ports
+    # since nginx is running under non-root user data-distillary
+    ports:
+      - "6666:8080"
+    healthcheck:
+      test: ["CMD", "curl", "--fail", "http://localhost:8080"]
+      interval: 1m30s
+      timeout: 10s
+      retries: 3
+      start_period: 40s
+    environment:
+      # UID and GID 1006 to match the data-distillary user by default.
+      # Execution on localhost should match the user running "docker-compose up"
+      - HOST_GID=${HOST_GID:-1001}
+      - HOST_UID=${HOST_UID:-1001}
```

Comment:
Set the default UID and GID both as 1006 to match the user account "data-distillery" on the PROD VM so we don't need to export each time during redeployment.

Reply:
done